aldryn_events/models.py

```diff
@@ -346,8 +346,8 @@ def get_name(self):
 
 class Registration(models.Model):
     SALUTATIONS = Choices(
-        ('SALUTATION_FEMALE', 'mrs', _('Frau')),
-        ('SALUTATION_MALE', 'mr', _('Herr')),
+        ('SALUTATION_FEMALE', 'mrs', ugettext('Ms.')),
```

Comment:
Unrelated, but this list is not exhaustive. We should have a standard list of salutations (maybe even a standard salutation field) and we should allow a blank value. Maybe @evildmp will have some ideas?